CHANGELOG.md

```diff
@@ -34,6 +35,103 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
+
+We expect that this release candidate is quite close to the final v1.1.0
+release. However, breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.0.0:
+
+## Standard Channel
+
+### GRPCRoute has Graduated to GA 🎉
+
+GRPCRoute has graduated to GA (v1) and is now part of the Standard Channel. If
+you are already using the experimental version GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: @gnossen
+
+### Mesh Support
+
+The standard for using Gateway API for Mesh has formally graduated to v1.0.
+
+Leading Contributors: @howardjohn, @keithmattix, @kflynn
+
+### Conformance Profiles and Reports
+
+The Conformance Reports API and the corresponding test suite have been graduated to GA. The Conformance report API has been expanded with the `mode` field (intended to specify the working mode of the implementation), and the `gatewayAPIChannel` (standard or experimental). The `gatewayAPIVersion` and `gatewayAPIChannel` are now filled in automatically by the suite machinery, along with a brief description of the testing outcome.
+The Reports have been reorganized in a more structured way, and the implementations can now add information on how the tests have been run and provide reproduction steps.
```

Comment:
I just noticed you seem to be 80-lining for some lines above, but not these? NBD but if you feel like updating it :shrug:

Reply:
These are just the result of me accepting GitHub suggestions. Once they're all in, I'll update the line length for everything.

Reply:
@robscott would you want to add anything about the work that @xtineskim completed?  It's user facing and it holds alot of meaning for some folks.